src/adk/api/remote_api.py

```diff
@@ -724,6 +724,41 @@ def experiments_list(self) -> List[ExperimentType]:
 
         return experiment_list
 
+    def __get_backend_type(self, backend_type: str) -> List[BackendTypeType]:
+        """
+        Get the backend info for the remote backend in the experiment data.
+
+        Returns:
+            0 or 1 backend types with this type
+        """
+        backend_types = self.__qne_client.list_backendtypes()
+        # "type" is required
+        backends = [backend for backend in backend_types if str(backend["name"]).lower() == backend_type.lower()]
+        if len(backends) > 1:
+            # QNE configuration error, should be solved in QNE
+            raise ExperimentValueError("More backend types with the same name configured in QNE")
+        return backends
+
+    def validate_experiment(self, experiment_data: ExperimentDataType, error_dict: ErrorDictType) -> None:
+        """
+        Validate if the remote backend settings are valid before experiment is created
+        """
+        backend_data = experiment_data["meta"]["backend"]
+        # "location" is required
+        if backend_data["location"].strip().lower() != "remote":
+            error_dict["error"].append("Backend location in experiment is not remote")
+        # "type" is required
+        backend_types = self.__get_backend_type(experiment_data["meta"]["backend"]["type"])
+        if len(backend_types) == 0:
+            error_dict["error"].append("Backend type in experiment is not a valid remote backend type "
+                                       "(names must match)")
+        else:
+            if not backend_types[0]["is_allowed"]:
+                error_dict["error"].append("The requested remote backend is not available for your current account "
+                                           "type, select a different backend.")
+            if backend_types[0]["status"] == "OFFLINE":
```

Comment:
does this make sense? BackendType has no status field

Reply:
Certainly it does have a property status. Otherwise this code would crash, not?

Reply:
Maybe the error messages talking about backends put you on the wrong leg (as the dutch say in good English:). Actually it would not make sense to make a difference between backend_type and backend for the end user. We keep that hidden as much as possible. To the end user we communicate about backends. Internally we have backend types and backends.